### ISSUE TITLE
UBports: installation of current stable OTA version

### ISSUE THREAD

**Issue by svenha** [gemian/gemian#22]
I found out that OTA updates for UBports (installed following this nice page: https://github.com/gemian/gemian/wiki/UBPorts ) work. Can the current stable OTA version be used as the initial artifacts on a new device? If so, how can these be found?

Currently, you have to install a developer version, switch channels in UBports, and update from within UBports to the stable OTA version. This works, but for new users this is more complex than needed.

**Comment by shymega** [comment]
This issue tracker is focused on Gemian itself, not UBPorts - [this](https://gitlab.com/ubports/community-ports/android9/planet-cosmocom/planet-cosmocom) repository over at GitLab is the best place to get answers - or the Planet Telegram group.

**Comment by svenha** [comment]
@shymega you are right. But I have so many open issues on this GitLab and wanted to avoid just another one ...

I also hoped to get some help from the Wiki page's author, @adamboardman :-)

**Comment by shymega** [comment]
Sure. @NotKit does the UBPorts stuff, but I don't think OTA is ready yet - @NotKit? I think we can ideally close this issue when your question has been answered.

**Comment by svenha** [comment]
OTA does work for the Cosmo Communicator. What is suboptimal is how to install a version that will do OTA :-)

**Comment by NotKit** [comment]
I think ubuntu.img is created as artifact of running flashable job on UBPorts repo, like https://gitlab.com/ubports/community-ports/android9/planet-cosmocom/planet-cosmocom/-/jobs/1649524521. I can create flashable artifacts manually for stable releases, but then we need to think where to host them (GitHub releases I guess) and link to them.

Ideally I'd want to see Cosmo supported by UBPorts Installer, but Cosmo dualboot setup makes it tricky (e. g. target boot partition can be named differently from device to device).

**Comment by svenha** [comment]
The stable OTA version was first (?) announced here: https://ubports.com/de/blog/ubports-blogs-nachrichten-1/post/ubuntu-touch-ota-19-release-3779 And the upgrade worked nicely. Who produced this version? Could not this version be used for an initial installation following this recipe: https://github.com/gemian/gemian/wiki/UBPorts

**Comment by NotKit** [comment]
The thing is UBPorts doesn't build official flashable images for most devices (except PinePhone?), expecting UBPorts Installer to be used for installation. That basically fetches desired OTA from system-image server and pushes it to device while that is in recovery mode.
